libraries/ESP8266SSDP/ESP8266SSDP.cpp

```diff
@@ -241,7 +246,7 @@ void SSDPClass::_send(ssdp_method_t method) {
                        _modelNumber,
                        _uuid,
                        (method == NONE) ? "ST" : "NT",
-                       _deviceType,
+                       (_st_is_uuid) ? _uuid : _deviceType,
```

Comment:
I moved the the prefix `uuid:` to the _uuid to avoid concatenating it in the line 249
alternative proposal would be
`(_st_is_uuid) ? String("uuid:" + String(_uuid)).c_str() : _deviceType,`
but this will introduce a bigger overhead I guess ..

do you have any other thoughts or alternatives I can use ??